apps/backend/app/services/job_service.py

```
@@ -12,125 +14,364 @@
+            return "Unknown Company"
         try:
-            structured_job: StructuredJobModel = StructuredJobModel.model_validate(
-                raw_output
-            )
-        except ValidationError as e:
-            logger.info(f"Validation error: {e}")
-            return None
-        return structured_job.model_dump(mode="json")
+            data = json.loads(company_profile_json)
+            return data.get("name", "Unknown Company")
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)